### ISSUE TITLE
creating files from file picker

### ISSUE THREAD

**Issue by gdkrmr** [helix-editor/helix#9071]
If I type a valid path into the file that does not exist it would be nice to create that file.

**Comment by TornaxO7** [comment]
Isn't it already possible with `:open <path>` and then `:w`?

**Comment by gdkrmr** [comment]
yes, this is possible. This isn't a reason to not being able to do it from the file picker though.

**Comment by aukeroorda** [comment]
One reason could be that you accidentally create files that you don't want. For me it would feel unintuitive/unexpected to create files from a picker

**Comment by gdkrmr** [comment]
I am coming from emacs helm where this functionality is normal. It creates a buffer and the actual file is created when the buffer is saved.
